packages/ssr-web/app/services/app-context.ts

```diff
@@ -83,6 +83,10 @@ export default class AppContext extends Service implements AppContextService {
         : window.location.search
     );
   }
+
+  getAbsolutePath(relativePath: string) {
+    return `https://${config.universalLinkDomain}${relativePath}`;
```

Comment:
What about this bothers you @backspace ?

Nit: Would go with throwing if the relative path doesn't start with / just in case folks edit or add on to this, but don't think it's a big deal.

Reply:
It’s just sad to me to use a hardcoded domain vs one derived from the request, but it’s not a big deal. I’ll add the leading `/`, thanks.